install_config/imagestreams_templates.adoc

```diff
@@ -296,6 +328,48 @@ database data will be lost if the database pod restarts for any reason.
 ====
 endif::[]
 
+ifdef::openshift-enterprise,openshift-origin[]
+----
+$ oc create -f $IMAGESTREAMDIR/image-streams-rhel7.json -n openshift
+----
+
+Alternatively, to create the core set of image streams that use the CentOS 7
+based images:
+
+----
+$ oc create -f $IMAGESTREAMDIR/image-streams-centos7.json -n openshift
+----
+
+It is not possible to create both the CentOS and RHEL sets of image streams
+because they use the same names. If you desire to have both sets of image
+streams available to users, either create one set in a different project, or
+edit one of the files and modify the image stream names to make them unique.
```

Comment:
This is some untouched content that's comin along for the commit-ride (see the ifdef it's within), but I'll look at possibly rewording now.